test/functional/test_framework/blocktools.py

```diff
@@ -65,6 +65,10 @@
 VERSIONBITS_LAST_OLD_BLOCK_VERSION = 4
 MIN_BLOCKS_TO_KEEP = 288
 
+REGTEST_N_BITS = 0x207fffff  # difficulty retargeting is disabled in REGTEST chainparams"
+
+def nbits_str(nbits):
+    return nbits.to_bytes(4, "big").hex()
```

Comment:
In commit "test: add nBits coverage" (ecb18b3405e4a61f872d03f667c167fb7c4a52cf)

`return f"{nbits:08x}"` would be a more direct conversion from number to string and would make the python code more consistent with the c++ code.

Reply:
Taken